sdk/documenttranslation/Azure.AI.DocumentTranslation/README.md

```diff
@@ -1,19 +1,488 @@
+```
 
 ### Prerequisites
+* An [Azure subscription][azure_sub].
+* An existing Cognitive Services or Translator resource.
+
+#### Create a Cognitive Services or Translator resource
+Translator supports both [multi-service and single-service access][cognitive_resource_portal]. Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Translator access only, create a Translator resource.
+
+You can create either resource using: 
+
+**Option 1:** [Azure Portal][cognitive_resource_portal].
+
+**Option 2:** [Azure CLI][cognitive_resource_cli]. 
+
+Below is an example of how you can create a Translator resource using the CLI:
+
+```PowerShell
+# Create a new resource group to hold the Translator resource -
+# if using an existing resource group, skip this step
+az group create --name <your-resource-name> --location <location>
+```
+
+```PowerShell
+# Create Translator
+az cognitiveservices account create \
+    --name <your-resource-name> \
+    --resource-group <your-resource-group-name> \
+    --kind TextTranslation \
+    --sku <sku> \
+    --location <location> \
+    --yes
+```
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
 
 ### Authenticate the client
+In order to interact with the Document Translation service, you'll need to create an instance of the [DocumentTranslationClient][documenttranslation_client_class] class. You will need an **endpoint**, and an **API key** to instantiate a client object.  For more information regarding authenticating with cognitive services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
+
+#### Get API Key
+You can get the `endpoint` and `API key` from the Cognitive Services resource or Translator resource information in the [Azure Portal][azure_portal].
+
+Alternatively, use the [Azure CLI][azure_cli] snippet below to get the API key from the Translator resource.
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+#### Create DocumentTranslationClient with API Key Credential
+Once you have the value for the API key, create an `AzureKeyCredential`. This will allow you to
+update the API key without creating a new client.
+
+With the value of the endpoint and an `AzureKeyCredential`, you can create the [DocumentTranslationClient][documenttranslation_client_class]:
+
+```C# Snippet:CreateDocumentTranslationClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var client = new DocumentTranslationClient(new Uri(endpoint), new AzureKeyCredential(apiKey));
+```
 
 ## Key concepts
 
+### DocumentTranslationClient
+A `DocumentTranslationClient` is the primary interface for developers using the Document Translation client library.  It provides both synchronous and asynchronous operations to access the Document Translation service.
```

Comment:
I feel like we should also add an Input section here and explain the general structure of the input. Maybe just describing in general what `TranslationConfiguration` is